src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -1,5 +1,7 @@
+import { strict as assert } from 'assert';
```

Comment:
I'd rather you throw an error instead of asserting (assertion error messages are not very useful).

Reply:
Done